posts/2023-02-27-introducing-new-conda.md

```diff
@@ -0,0 +1,53 @@
+```
+* `analysis3/22.07`, `analysis3/22.10` and `analysis3/23.01` environments are available - they are exact duplicates of the corresponding existing `analysis3` environments.
+* New module for ARE - `conda_concept/are`.
+![conda-are-settings](../images/conda-are-settings.png)
+* Some catches:
+    * Can't use direct path to interpreter, must use path to the 'scripts' directory instead if a full path is needed (e.g. in `#!/g/data/hh5/.../python3` shbang lines in scripts)
+    * `conda activate` does not work, must load the module.
+* Any problems or questions, contact cws_help@nci.org.au, or leave an issue on the [github page](https://github.com/coecms/cms-conda-singularity/issues).
+
+## Usage
+Usage of the new `conda_concept/analysis3` environments is very similar to the current `conda/analysis3` environments. On the Gadi command line, or in PBS scripts, run
+```
+$ module use /g/data/hh5/public/modules
+$ module load conda_concept/analysis3
+```
+The version naming scheme is carried over from the current `conda/analysis3` environments. The `analysis3-unstable` environment will always be an alias for the most recent environment, and will continue to be updated at roughly 3 monthly intervals. `analysis3` will always be an alias for the next-most recent environment, referred to as the "stable" environment. `module load conda_concept` will always be configured to load the "stable" environment. Specific versions can also be loaded with e.g. `module load conda_concept/analysis3-23.01`. There is also a new module, `conda_concept/are`, designed specifically for use with `ipython` and `jupyter` notebooks. This module loads the "stable" `analysis3` environment, but manipulates the environment such that all of the `analysis3` environments can be used as the kernel for a notebook without having to restart the `jupyter` session.
+
+On installation or update, a symlink to the launcher script corresponding to every command within the conda environment is created in a separate scripts directory. The launcher script will determine which command needs to be run from the conda environment based on the name of the symlink. For example, running the `jupyter` symlink in the `cms_conda_scripts/analysis3-23.01.d/bin/` directory is the equivalent of running `cms_conda/envs/analysis3-23.10/bin/jupyter` from within the container. The launcher script can also run arbitrary commands. For example, to launch a shell within the container, run `launcher.sh bash`. It is this scripts directory that is placed in `$PATH`, and launches Singularity only when necessary. The downside of this is that `singularity` can be expensive to launch. If your workflow requires launching applications from the `analysis3` environment in quick succession, it may be worth re-factoring the workflow such that it runs from within the container using `launcher.sh`. The typical `analysis3` workflow comprising python scripts or jupyter notebooks is not affected by this, and can be run as normal.

```

Comment:
For someone not very experienced this information might looked scary and complicated. We should have the "The typical `analysis3` workflow comprising python scripts or jupyter notebooks is not affected by this, and can be run as normal." reassurance at the start and highlight at first in which use cases this detailed info is useful. Hope this makes sense